default-config.json

```diff
@@ -183,13 +187,17 @@
       "prefix": "https://cadenceradio.com",
       "search": "/api/search",
       "request": "/api/request/id",
-      "library": "/api/aria1/library",
+      "library": "NOT IMPLEMENTED BY UPSTREAM",
       "nowplaying": "/api/nowplaying/metadata",
+      "history": "/api/history",
+      "listenurl": "/api/listenurl",
       "key": false
     },
     "stream": {
-      "prefix": "https://stream.cadenceradio.com",
-      "stream": "/cadence1"
+      "automatic": true,
+      "protocol": "https",
```

Comment:
The Cadence API reference says we choose this - I decided to enforce that fallback and primary both use the same protocol, but not to enforce that that protocol matches the main API prefix.

I'm open to disagreement with that choice though.